examples/magento-graphcms/pages/customer/account/createPassword/index.tsx

```diff
@@ -20,7 +20,11 @@ function CustomerAccountCreatePasswordPage() {
 
   return (
     <>
-      <PageMeta title={i18n._(/* i18n */ 'Create new password')} metaRobots={['noindex']} />
+      <PageMeta
+        title={i18n._(/* i18n */ 'Create new password')}
+        metaDescription={i18n._(/* i18n */ 'Create new password')}
```

Comment:
Page is not indexable, metaDescription is useless. Adding this change creates useless churn